docs/device-data/data-types/basal/automated.md

```diff
@@ -1,13 +1,15 @@
 <!-- omit in toc -->
 # Automated Basals (`automated`)
 
-<!-- omit in toc -->
-## Table of Contents
+## Delivery Type (`deliveryType`)
```

Comment:
It might make sense to promote (demote?) this `deliveryType` to a partial as well?

Reply:
Oh nvm, I see it differs with a different enum string...